topic-operator/src/main/java/io/strimzi/operator/topic/Session.java

```diff
@@ -221,8 +223,18 @@ public void handle(Long oldTimerId) {
                         if (!stopped) {
                             timerId = null;
                             boolean isInitialReconcile = oldTimerId == null;
+                            Timer.Sample reconciliationTimerSample = Timer.start(topicOperator.getMetrics().meterRegistry());
```

Comment:
In CO and UO, the timer as well as some of the counters are measured per resource. So if you have 10 Kafka topics, you will get 10 measurements. This looks to me like it really measures only the reconcilaAll. Or do I misunderstand what this code in the operator does?